CHANGELOG.md

```diff
@@ -2,92 +2,62 @@
 
 [npm history][1]
 
-[1]: https://www.npmjs.com/package/nodejs-spanner?activeTab=versions
```

Comment:
This doesn't look right *scratches head* it looks like the v2 changelog is getting written over.  No idea how that happened :(

Reply:
oh no, should it be additive?

Reply:
Yup.  Idea is to have an entry in here for every release.

Reply:
To be clear, the tool should do this auto-magically.  I have no idea how you got in this state :/

Reply:
Thats my mistake, when the PR was made it looked to be based off of my forks master state (which is super out of date) and I rebased so I wouldn't have to re-sort the changelog.